### ISSUE TITLE
Intermitted malformed request bodys

### ISSUE THREAD

**Issue by dustinblackman** [atom/electron#2890]
Electron: __0.33.1__
Node: __4.1.0__
OS: __OSX 10.10.3__

I'm currently using Electron to collect data from a website (with permission, of course), and since moving from 0.30.5/2.1.3 to 0.33.1/4.1.0 I seem to be getting a large amount of malformed request bodies that don't happen outside of Electron.

Example:
Using the latest versions of the following modules, I make a request for each name. Every page has a `script` tag that contains `matchupData.` On every run however some of the names will fail, and it's never the same ones in a group.

```coffeescript
async = require 'async'
cheerio = require 'cheerio'
request = require 'request'
_ = require 'lodash'

champs = ["Aatrox","Ahri","Akali","Alistar","Amumu","Anivia","Annie","Ashe","Azir","Bard",
  "Blitzcrank","Brand","Braum","Caitlyn","Cassiopeia","Chogath","Corki","Darius","Diana",
  "Draven","DrMundo","Ekko","Elise","Evelynn","Ezreal","FiddleSticks","Fiora","Fizz",
  "Galio","Gangplank","Garen","Gnar","Gragas","Graves","Hecarim","Heimerdinger",
  "Irelia","Janna","JarvanIV","Jax","Jayce","Jinx","Kalista","Karma","Karthus",
  "Kassadin","Katarina","Kayle","Kennen","Khazix","KogMaw","Leblanc","LeeSin",
  "Leona","Lissandra","Lucian","Lulu","Lux","Malphite","Malzahar","Maokai","MasterYi",
  "MissFortune","MonkeyKing","Mordekaiser","Morgana","Nami","Nasus","Nautilus","Nidalee",
  "Nocturne","Nunu","Olaf","Orianna","Pantheon","Poppy","Quinn","Rammus","RekSai",
  "Renekton","Rengar","Riven","Rumble","Ryze","Sejuani","Shaco","Shen","Shyvana","Singed",
  "Sion","Sivir","Skarner","Sona","Soraka","Swain","Syndra","TahmKench","Talon","Taric",
  "Teemo","Thresh","Tristana","Trundle","Tryndamere","TwistedFate","Twitch","Udyr","Urgot",
  "Varus","Vayne","Veigar","Velkoz","Vi","Viktor","Vladimir","Volibear","Warwick","Xerath",
  "XinZhao","Yasuo","Yorick","Zac","Zed","Ziggs","Zilean","Zyra"]

async.eachLimit champs, 3, (champ, step) ->
  request "http://champion.gg/champion/#{champ}", (err, res, body) ->
    $c = cheerio.load(body)
    script_tag = $c('script:contains("matchupData.")').text()
    console.log("#{champ} FAILED") if !script_tag
    step()
, ->
  console.log('Done')
```

When failed is logged, the body returned looks broken like this. (Half the pages data is missing, part of which is the end of the comment line for shim.

```html
<!doctype html>
<html lang="en" ng-app="championPage"><head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="description" content="LoL Statistics, Builds, Runes, Masteries, Skill Orders, Counters and Matchups for Master Yi when played Jungle. Statistics include Master Yi&#39;s Win Rate, Play Rate and Ban Rate. Counters include who Master Yi Jungle is Strong or Weak Against.">
  <title>Champion.gg - Master Yi Jungle Stats, Builds, Runes, Masteries and Counters</title>
  <!-- Bootstrap core CSS -->
  <link href="/dist/css/bootstrap.min.css" rel="stylesheet">
  
    <link href="/css/master.min.css?v=5.18.10.759802" rel="stylesheet">
  
  
  <script>
  var matchupData = {};
  </script>
  <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
  <!--[if lt IE 9]>
  <script src="https://oss.maxcdn600c0edc56975bc1ea8b73a"},{"games":615,"statScore":6.05,"winRate":50.73,"winRateChange":1.57,"key":"Hecarim","_id":"5600c0edc56975bc1ea8b739"},{"games":229,"statScore":2.75,"winRate":48.03,"winRateChange":-1.13,"key":"Tryndamere","_id":"5600c0edc56975bc1ea8b738"},{"games":1210,"statScore":5.94,"winRate":51.32,"winRateChange":2.16,"key":"RekSai","_id":"5600c0edc56975bc1ea8b737"},{"games":1345,"statScore":4.27,"winRate":47.13,"winRateChange":-2.03,"key":"Nocturne","_id":"5600c0edc56975bc1ea8b736"},{"games":1379,"statScore":3.24,"winRate":45.75,"winRateChange":-3.41,"key":"Shaco","_id":"5600c0edc56975bc1ea8b735"},{"games":1627,"statScore":3.99,"winRate":44.19,"winRateChange":-4.97,"key":"Warwick","_id":"5600c0edc56975bc1ea8b734"},{"games":1491,"statScore":3.43,"winRate":45.67,"winRateChange":-3.49,"key":"Udyr","_id":"5600c0edc56975bc1ea8b733"},{"games":1544,"statScore":2.88,"winRate":48.7,"winRateChange":-0.46,"key":"Khazix","_id":"5600c0edc56975bc1ea8b732"},{"games":741,"statScore":4.16,"winRate":47.09,"winRateChange":-2.07,"key":"Pantheon","_id":"5600c0edc56975bc1ea8b731"},{"games":1709,"statScore":4.25,"winRate":45.93,"winRateChange":-3.23,"key":"Skarner","_id":"5600c0edc56975bc1ea8b730"},{"games":1329,"statScore":6.11,"winRate":52.06,"winRateChange":2.9,"key":"Nidalee","_id":"5600c0edc56975bc1ea8b72f"},{"games":1738,"statScore":5.57,"winRate":50,"winRateChange":0.84,"key":"Evelynn","_id":"5600c0edc56975bc1ea8b72e"},{"games":406,"statScore":8.39,"winRate":55.91,"winRateChange":6.75,"key":"Olaf","_id":"5600c0edc56975bc1ea8b72d"},{"games":1375,"statScore":6.93,"winRate":54.32,"winRateChange":5.16,"key":"Gragas","_id":"5600c0edc56975bc1ea8b72c"},{"games":2148,"statScore":4.74,"winRate":45.48,"winRateChange":-3.68,"key":"Amumu","_id":"5600c0edc56975bc1ea8b72b"},{"games":1318,"statScore":7.7,"winRate":55.91,"winRateChange":6.75,"key":"JarvanIV","_id":"5600c0edc56975bc1ea8b72a"},{"games":405,"statScore":5.48,"winRate":49.38,"winRateChange":0.22,"key":"TahmKench","_id":"5600c0edc56975bc1ea8b729"},{"games":1012,"statScore":4.97,"winRate":49.01,"winRateChange":-0.15,"key":"Jax","_id":"5600c0edc56975bc1ea8b728"},{"games":2477,"statScore":4.31,"winRate":49.21,"winRateChange":0.05,"key":"Shyvana","_id":"5600c0edc56975bc1ea8b727"},{"games":1093,"statScore":3.94,"winRate":45.83,"winRateChange":-3.33,"key":"MonkeyKing","_id":"5600c0edc56975bc1ea8b726"},{"games":2585,"statScore":3.96,"winRate":46.73,"winRateChange":-2.43,"key":"Elise","_id":"5600c0edc56975bc1ea8b725"},{"games":2925,"statScore":5.13,"winRate":48.95,"winRateChange":-0.21,"key":"Vi","_id":"5600c0edc56975bc1ea8b724"},{"games":488,"statScore":6.49,"winRate":50.4,"winRateChange":1.24,"key":"Nautilus","_id":"5600c0edc56975bc1ea8b723"},{"games":3364,"statScore":4.2,"winRate":46.31,"winRateChange":-2.85,"key":"XinZhao","_id":"5600c0edc56975bc1ea8b722"},{"games":737,"statScore":4.87,"winRate":47.62,"winRateChange":-1.54,"key":"Fizz","_id":"5600c0edc56975bc1ea8b721"},{"games":2758,"statScore":6.42,"winRate":51.7,"winRateChange":2.54,"key":"Rengar","_id":"5600c0edc56975bc1ea8b720"},{"games":557,"statScore":7.41,"winRate":52.06,"winRateChange":2.9,"key":"Malphite","_id":"5600c0edc56975bc1ea8b71f"},{"games":1819,"statScore":4.94,"winRate":47.99,"winRateChange":-1.17,"key":"Diana","_id":"5600c0edc56975bc1ea8b71e"},{"games":2331,"statScore":5.61,"winRate":49.54,"winRateChange":0.38,"key":"Ekko","_id":"5600c0edc56975bc1ea8b71d"},{"games":7004,"statScore":5.34,"winRate":50.69,"winRateChange":1.53,"key":"LeeSin","_id":"5600c0edc56975bc1ea8b71c"}],"unique":null,"runes":{"highestWinPercent":{"games":757,"winPercent":55.21,"runes":[{"id":5245,"number":9,"name":"Greater Mark of Attack Damage","img":"r_1_3.png","description":"+0.95 attack damage","_id":"5600c0edc56975bc1ea8b691"},{"id":5277,"number":9,"name":"Greater Glyph of Attack Speed","img":"b_3_3.png","description":"+0.64% attack speed","_id":"5600c0edc56975bc1ea8b690"},{"id":5317,"number":9,"name":"Greater Seal of Armor","img":"y_1_3.png","description":"+1 armor","_id":"5600c0edc56975bc1ea8b68f"},{"id":5337,"number":3,"name":"Greater Quintessence of Attack Speed","img":"bl_3_3.png","description":"+4.5% attack speed","_id":"5600c0edc56975bc1ea8b68e"}]},"mostGames":{"games":5569,"winPercent":48.82,"runes":[{"id":5245,"number":9,"name":"Greater Mark of Attack Damage","img":"r_1_3.png","description":"+0.95 attack damage","_id":"5600c0edc56975bc1ea8b695"},{"id":5289,"number":9,"name":"Greater Glyph of Magic Resist","img":"b_3_3.png","description":"+1.34 magic resist","_id":"5600c0edc56975bc1ea8b694"},{"id":5317,"number":9,"name":"Greater Seal of Armor","img":"y_1_3.png","description":"+1 armor","_id":"5600c0edc56975bc1ea8b693"},{"id":5337,"number":3,"name":"Greater Quintessence of Attack Speed","img":"bl_3_3.png","description":"+4.5% attack speed","_id":"5600c0edc56975bc1ea8b692"}]}},"masteries":{"highestWinPercent":{"games":140,"winPercent":58.57,"masteries":[{"tree":"Offense","total":21,"_id":"5600c0edc56975bc1ea8b6c2","data":{"row6":[{"points":0,"mastery":null,"_id":"5600c0edc56975bc1ea8b6c4"},{"points":1,"mastery":4162,"_id":"5600c0edc56975bc1ea8b6c3"}],"row5":[{"points":1,"mastery":4151,"_id":"5600c0edc56975bc1ea8b6c8"},{"points":3,"mastery":4152,"_id":"5600c0edc56975bc1ea8b6c7"},{"points":0,"mastery":null,"_id":"5600c0edc56975bc1ea8b6c6"},{"points":0,"mastery":4154,"_id":"5600c0edc56975bc1ea8b6c5"}],"row4":[{"points":0,"mastery":4141,"_id":"5600c0edc56975bc1ea8b6cc"},{"points":3,"mastery":4142,"_id":"5600c0edc56975bc1ea8b6cb"},{"points":0,"mastery":4143,"_id":"5600c0edc56975bc1ea8b6ca"},{"points":1,"mastery":4144,"_id":"5600c0edc56975bc1ea8b6c9"}],"row3":[{"points":0,"mastery":4131,"_id":"5600c0edc56975bc1ea8b6d0"},{"points":1,"mastery":4132,"_id":"5600c0edc56975bc1ea8b6cf"},{"points":0,"mastery":4133,"_id":"5600c0edc56975bc1ea8b6ce"},{"points":3,"mastery":4134,"_id":"5600c0edc56975bc1ea8b6cd"}],"row2":[{"points":0,"mastery":4121,"_id":"5600c0edc56975bc1ea8b6d4"},{"points":3,"mastery":4122,"_id":"5600c0edc56975bc1ea8b6d3"},{"points":0,"mastery":4123,"_id":"5600c0edc56975bc1ea8b6d2"},{"points":0,"mastery":4124,"_id":"5600c0edc56975bc1ea8b6d1"}],"row1":[{"points":1,"mastery":4111,"_id":"5600c0edc56975bc1ea8b6d8"},{"points":4,"mastery":4112,"_id":"5600c0edc56975bc1ea8b6d7"},{"points":0,"mastery":4113,"_id":"5600c0edc56975bc1ea8b6d6"},{"points":0,"mastery":4114,"_id":"5600c0edc56975bc1ea8b6d5"}]}},{"tree":"Defense","total":6,"_id":"5600c0edc56975bc1ea8b6ac","data":{"row6":[{"points":0,"mastery":null,"_id":"5600c0edc56975bc1ea8b6ae"},{"points":0,"mastery":4262,"_id":"5600c0edc56975bc1ea8b6ad"}],"row5":[{"points":0,"mastery":4251,"_id":"5600c0edc56975bc1ea8b6b1"},{"points":0,"mastery":4252,"_id":"5600c0edc56975bc1ea8b6b0"},{"points":0,"mastery":4253,"_id":"5600c0edc56975bc1ea8b6af"}],"row4":[{"points":0,"mastery":4241,"_id":"5600c0edc56975bc1ea8b6b5"},{"points":0,"mastery":4242,"_id":"5600c0edc56975bc1ea8b6b4"},{"points":0,"mastery":4243,"_id":"5600c0edc56975bc1ea8b6b3"},{"points":0,"mastery":4244,"_id":"5600c0edc56975bc1ea8b6b2"}],"row3":[{"points":0,"mastery":4231,"_id":"5600c0edc56975bc1ea8b6b9"},{"points":0,"mastery":4232,"_id":"5600c0edc56975bc1ea8b6b8"},{"points":0,"mastery":4233,"_id":"5600c0edc56975bc1ea8b6b7"},{"points":0,"mastery":4234,"_id":"5600c0edc56975bc1ea8b6b6"}],"row2":[{"points":1,"mastery":4221,"_id":"5600c0edc56975bc1ea8b6bd"},{"points":0,"mastery":4222,"_id":"5600c0edc56975bc1ea8b6bc"},{"points":0,"mastery":null,"_id":"5600c0edc56975bc1ea8b6bb"},{"points":1,"mastery":4224,"_id":"5600c0edc56975bc1ea8b6ba"}],"row1":[{"points":2,"mastery":4211,"_id":"5600c0edc56975bc1ea8b6c1"},{"points":0,"mastery":4212,"_id":"5600c0edc56975bc1ea8b6c0"},{"points":0,"mastery":4213,"_id":"5600c0edc56975bc1ea8b6bf"},{"points":2,"mastery":4214,"_id":"5600c0edc56975bc1ea8b6be"}]}},{"tree":"Utility","total":3,"_id":"5600c0edc56975bc1ea8b696","data":{"row6":[{"points":0,"mastery":null,"_id":"5600c0edc56975bc1ea8b698"},{"points":0,"mastery":4362,"_id":"5600c0edc56975bc1ea8b697"}],"row5":[{"points":0,"mastery":null,"_id":"5600c0edc56975bc1ea8b69b"},{"points":0,"mastery":4352,"_id":"5600c0edc56975bc1ea8b69a"},{"points":0,"mastery":4353,"_id":"5600c0edc56975bc1ea8b699"}],"row4":[{"points":0,"mastery":4341,"_id":"5600c0edc56975bc1ea8b69f"},{"points":0,"mastery":4342,"_id":"5600c0edc56975bc1ea8b69e"},{"points":0,"mastery":4343,"_id":"5600c0edc56975bc1ea8b69d"},{"points":0,"mastery":4344,"_id":"5600c0edc56975bc1ea8b69c"}],"row3":[{"points":0,"mastery":4331,"_id":"5600c0edc56975bc1ea8b6a3"},{"points":0,"mastery":4332,"_id":"5600c0edc56975bc1ea8b6a2"},{"points":0,"mastery":4333,"_id":"5600c0edc56975bc1ea8b6a1"},{"points":0,"mastery":4334,"_id":"5600c0edc56975bc1ea8b6a0"}],"row2":[{"points":0,"mastery":null,"_id":"5600c0edc56975bc1ea8b6a7"},{"points":0,"mastery":4322,"_id":"5600c0edc56975bc1ea8b6a6"},{"points":0,"mastery":4323,"_id":"5600c0edc56975bc1ea8b6a5"},{"points":0,"mastery":4324,"_id":"5600c0edc56975bc1ea8b6a4"}],"row1":[{"points":0,"mastery":4311,"_id":"5600c0edc56975bc1ea8b6ab"},{"points":3,"mastery":4312,"_id":"5600c0edc56975bc1ea8b6aa"},{"points":0,"mastery":4313,"_id":"5600c0edc56975bc1ea8b6a9"},{"points":0,"mastery":4314,"_id":"5600c0edc56975bc1ea8b6a8"}]}}]},"mostGames":{"games":2473,"winPercent":52.72,"masteries":[{"tree":"Offense","total":21,"_id":"5600c0edc56975bc1ea8b705","data":{"row6":[{"points":0,"mastery":null,"_id":"5600c0edc56975bc1ea8b707"},{"points":1,"mastery":4162,"_id":"5600c0edc56975bc1ea8b706"}],"row5":[{"points":0,"mastery":4151,"_id":"5600c0edc56975bc1ea8b70b"},{"points":3,"mastery":4152,"_id":"5600c0edc56975bc1ea8b70a"},{"points":0,"mastery":null,"_id":"5600c0edc56975bc1ea8b709"},{"points":0,"mastery":4154,"_id":"5600c0edc56975bc1ea8b708"}],"row4":[{"points":0,"mastery":4141,"_id":"5600c0edc56975bc1ea8b70f"},{"points":3,"mastery":4142,"_id":"5600c0edc56975bc1ea8b70e"},{"points":0,"mastery":4143,"_id":"5600c0edc56975bc1ea8b70d"},{"points":1,"mastery":4144,"_id":"5600c0edc56975bc1ea8b70c"}],"row3":[{"points":0,"mastery":4131,"_id":"5600c0edc56975bc1ea8b713"},{"points":1,"mastery":4132,"_id":"5600c0edc56975bc1ea8b712"},{"points":0,"mastery":4133,"_id":"5600c0edc56975bc1ea8b711"},{"points":3,"mastery":4134,"_id":"5600c0edc56975bc1ea8b710"}],"row2":[{"points":0,"mastery":4121,"_id":"5600c0edc56975bc1ea8b717"},{"points":3,"mastery":4122,"_id":"5600c0edc56975bc1ea8b716"},{"points":0,"mastery":4123,"_id":"5600c0edc56975bc1ea8b715"},{"points":0,"mastery":4124,"_id":"5600c0edc56975bc1ea8b714"}],"row1":[{"points":1,"mastery":4111,"_id":"5600c0edc56975bc1ea8b71b"},{"points":4,"mastery":4112,"_id":"5600c0edc56975bc1ea8b71a"},{"points":0,"mastery":4113,"_id":"5600c0edc56975bc1ea8b719"},{"points":1,"mastery":4114,"_id":"5600c0edc56975bc1ea8b718"}]}},{"tree":"Defense","total":9,"_id":"5600c0edc56975bc1ea8b6ef","data":{"row6":[{"points":0,"mastery":null,"_id":"5600c0edc56975bc1ea8b6f1"},{"points":0,"mastery":4262,"_id":"5600c0edc56975bc1ea8b6f0"}],"row5":[{"points":0,"mastery":4251,"_id":"5600c0edc56975bc1ea8b6f4"},{"points":0,"mastery":4252,"_id":"5600c0edc56975bc1ea8b6f3"},{"points":0,"mastery":4253,"_id":"5600c0edc56975bc1ea8b6f2"}],"row4":[{"points":0,"mastery":4241,"_id":"5600c0edc56975bc1ea8b6f8"},{"points":0,"mastery":4242,"_id":"5600c0edc56975bc1ea8b6f7"},{"points":0,"mastery":4243,"_id":"5600c0edc56975bc1ea8b6f6"},{"points":0,"mastery":4244,"_id":"5600c0edc56975bc1ea8b6f5"}],"row3":[{"points":0,"mastery":4231,"_id":"5600c0edc56975bc1ea8b6fc"},{"points":1,"mastery":4232,"_id":"5600c0edc56975bc1ea8b6fb"},{"points":0,"mastery":4233,"_id":"5600c0edc56975bc1ea8b6fa"},{"points":0,"mastery":4234,"_id":"5600c0edc56975bc1ea8b6f9"}],"row2":[{"points":1,"mastery":4221,"_id":"5600c0edc56975bc1ea8b700"},{"points":3,"mastery":4222,"_id":"5600c0edc56975bc1ea8b6ff"},{"points":0,"mastery":null,"_id":"5600c0edc56975bc1ea8b6fe"},{"points":0,"mastery":4224,"_id":"5600c0edc56975bc1ea8b6fd"}],"row1":[{"points":2,"mastery":4211,"_id":"5600c0edc56975bc1ea8b704"},{"points":0,"mastery":4212,"_id":"5600c0edc56975bc1ea8b703"},{"points":0,"mastery":4213,"_id":"5600c0edc56975bc1ea8b702"},{"points":2,"mastery":4214,"_id":"5600c0edc56975bc1ea8b701"}]}},{"tree":"Utility","total":0,"_id":"5600c0edc56975bc1ea8b6d9","data":{"row6":[{"points":0,"mastery":null,"_id":"5600c0edc56975bc1ea8b6db"},{"points":0,"mastery":4362,"_id":"5600c0edc56975bc1ea8b6da"}],"row5":[{"points":0,"mastery":null,"_id":"5600c0edc56975bc1ea8b6de"},{"points":0,"mastery":4352,"_id":"5600c0edc56975bc1ea8b6dd"},{"points":0,"mastery":4353,"_id":"5600c0edc56975bc1ea8b6dc"}],"row4":[{"points":0,"mastery":4341,"_id":"5600c0edc56975bc1ea8b6e2"},{"points":0,"mastery":4342,"_id":"5600c0edc56975bc1ea8b6e1"},{"points":0,"mastery":4343,"_id":"5600c0edc56975bc1ea8b6e0"},{"points":0,"mastery":4344,"_id":"5600c0edc56975bc1ea8b6df"}],"row3":[{"points":0,"mastery":4331,"_id":"5600c0edc56975bc1ea8b6e6"},{"points":0,"mastery":4332,"_id":"5600c0edc56975bc1ea8b6e5"},{"points":0,"mastery":4333,"_id":"5600c0edc56975bc1ea8b6e4"},{"points":0,"mastery":4334,"_id":"5600c0edc56975bc1ea8b6e3"}],"row2":[{"points":0,"mastery":null,"_id":"5600c0edc56975bc1ea8b6ea"},{"points":0,"mastery":4322,"_id":"5600c0edc56975bc1ea8b6e9"},{"points":0,"mastery":4323,"_id":"5600c0edc56975bc1ea8b6e8"},{"points":0,"mastery":4324,"_id":"5600c0edc56975bc1ea8b6e7"}],"row1":[{"points":0,"mastery":4311,"_id":"5600c0edc56975bc1ea8b6ee"},{"points":0,"mastery":4312,"_id":"5600c0edc56975bc1ea8b6ed"},{"points":0,"mastery":4313,"_id":"5600c0edc56975bc1ea8b6ec"},{"points":0,"mastery":4314,"_id":"5600c0edc56975bc1ea8b6eb"}]}}]}},"skills":{"highestWinPercent":{"games":3451,"winPercent":50.53,"order":["1","3","1","2","1","4","1","3","1","3","4","3","3","2","2","4","2","2"]},"mostGames":{"games":26611,"winPercent":49.64,"order":["1","3","2","1","1","4","1","3","1","3","4","3","3","2","2","4","2","2"]},"skillInfo":[{"name":"Alpha Strike","img":"AlphaStrike.png","key":"Q","_id":"5600c0edc56975bc1ea8b68d"},{"name":"Meditate","img":"Meditate.png","key":"W","_id":"5600c0edc56975bc1ea8b68c"},{"name":"Wuju Style","img":"WujuStyle.png","key":"E","_id":"5600c0edc56975bc1ea8b68b"},{"name":"Highlander","img":"Highlander.png","key":"R","_id":"5600c0edc56975bc1ea8b68a"}]},"summoners":{"highestWinPercent":{"games":11498,"winPercent":49.51,"summoner2":{"name":"Teleport","url":"SummonerTeleport.png"},"summoner1":{"name":"Smite","url":"SummonerSmite.png"}},"mostGames":{"games":40098,"winPercent":48.87,"summoner2":{"name":"Smite","url":"SummonerSmite.png"},"summoner1":{"name":"Flash","url":"SummonerFlash.png"}}},"trinkets":[{"games":54721,"winPercent":49.26,"_id":"5600c0edc56975bc1ea8b689","item":{"id":3340,"name":"Warding Totem (Trinket)"}},{"games":472,"winPercent":49.78,"_id":"5600c0edc56975bc1ea8b688","item":{"id":3341,"name":"Sweeping Lens (Trinket)"}},{"games":50,"winPercent":50,"_id":"5600c0edc56975bc1ea8b687","item":{"id":3342,"name":"Scrying Orb (Trinket)"}}],"firstItems":{"highestWinPercent":{"games":54226,"winPercent":49.24,"items":[{"id":1039,"name":"Hunter's Machete","_id":"5600c0edc56975bc1ea8b683"},{"id":2003,"name":"Health Potion","_id":"5600c0edc56975bc1ea8b682"},{"id":2003,"name":"Health Potion","_id":"5600c0edc56975bc1ea8b681"}]},"mostGames":{"games":54226,"winPercent":49.24,"items":[{"id":1039,"name":"Hunter's Machete","_id":"5600c0edc56975bc1ea8b686"},{"id":2003,"name":"Health Potion","_id":"5600c0edc56975bc1ea8b685"},{"id":2003,"name":"Health Potion","_id":"5600c0edc56975bc1ea8b684"}]}},"items":{"highestWinPercent":{"games":2302,"winPercent":51.33,"items":[{"id":3718,"name":"Enchantment: Devourer","_id":"5600c0edc56975bc1ea8b67a"},{"id":3153,"name":"Blade of the Ruined King","_id":"5600c0edc56975bc1ea8b679"},{"id":3006,"name":"Berserker's Greaves","_id":"5600c0edc56975bc1ea8b678"},{"id":3142,"name":"Youmuu's Ghostblade","_id":"5600c0edc56975bc1ea8b677"},{"id":3031,"name":"Infinity Edge","_id":"5600c0edc56975bc1ea8b676"},{"id":3035,"name":"Last Whisper","_id":"5600c0edc56975bc1ea8b675"}]},"mostGames":{"games":3324,"winPercent":46.99,"items":[{"id":3718,"name":"Enchantment: Devourer","_id":"5600c0edc56975bc1ea8b680"},{"id":3006,"name":"Berserker's Greaves","_id":"5600c0edc56975bc1ea8b67f"},{"id":3153,"name":"Blade of the Ruined King","_id":"5600c0edc56975bc1ea8b67e"},{"id":3142,"name":"Youmuu's Ghostblade","_id":"5600c0edc56975bc1ea8b67d"},{"id":3035,"name":"Last Whisper","_id":"5600c0edc56975bc1ea8b67c"},{"id":3031,"name":"Infinity Edge","_id":"5600c0edc56975bc1ea8b67b"}]}},"dmgComposition":{"physicalDmg":69.6,"magicDmg":10.8,"trueDmg":19.6},"patchWin":[47.92,48.95,48.57,49.9,49.16],"experienceSample":[21.4,21.86,29.22,14.01,12.55],"experienceRate":[46.07,48.01,49.68,50.95,52.95],"gameLength":[44.39,48.34,50.99,50.37,50.17],"patchPlay":[9.08,9.3,7.68,9.07,12.59],"championMatrix":[84.13325354079387,152.26284089007868,195,15,13.502538071066027,116.2827666163855,96.23091442561064],"overallPosition":{"position":4,"change":2},"general":[{"title":"Win Rate","titleLink":"winPercent","val":"49.16","position":32,"change":-5,"_id":"5600c0edc56975bc1ea8b674"},{"title":"Play Rate","titleLink":"playPercent","val":"12.59","position":2,"change":4,"_id":"5600c0edc56975bc1ea8b673"},{"title":"Ban Rate","titleLink":"banRate","val":"8.96","position":7,"change":2,"_id":"5600c0edc56975bc1ea8b672"},{"title":"Playerbase Average Games Played","titleLink":"experience","val":"77.01","position":17,"change":0,"_id":"5600c0edc56975bc1ea8b671"},{"title":"Gold Earned","titleLink":"goldEarned","val":"13179","position":1,"change":0,"_id":"5600c0edc56975bc1ea8b670"},{"title":"Kills","titleLink":"kills","val":"8.78","position":1,"change":0,"_id":"5600c0edc56975bc1ea8b66f"},{"title":"Deaths","titleLink":"deaths","val":"7.33","position":44,"change":0,"_id":"5600c0edc56975bc1ea8b66e"},{"title":"Assists","titleLink":"assists","val":"5.93","position":44,"change":0,"_id":"5600c0edc56975bc1ea8b66d"},{"title":"Damage Dealt","titleLink":"totalDamageDealtToChampions","val":"19131","position":3,"change":-1,"_id":"5600c0edc56975bc1ea8b66c"},{"title":"Damage Taken","titleLink":"totalDamageTaken","val":"31097","position":24,"change":1,"_id":"5600c0edc56975bc1ea8b66b"},{"title":"Team Jungle CS","titleLink":"neutralMinionsKilledTeamJungle","val":"73.33","position":3,"change":0,"_id":"5600c0edc56975bc1ea8b66a"},{"title":"Counter Jungle CS","titleLink":"neutralMinionsKilledEnemyJungle","val":"11.01","position":9,"change":-1,"_id":"5600c0edc56975bc1ea8b669"}]};
  matchupData.patchHistory = ["5.14","5.15","5.16","5.17","5.18"];
  /* Has general data on the chosen champion */
</script>

	<script src="/js/master.min.js?v=5.18.10.759802"></script>


	

<div class="footer-attr">
Champion.gg isn't endorsed by Riot Games and doesn't reflect the views or opinions of Riot Games or anyone officially involved in producing or managing League of Legends. League of Legends and Riot Games are trademarks or registered trademarks of Riot Games, Inc. League of Legends © Riot Games, Inc.
</div>

<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-64942437-1', 'auto');
  ga('send', 'pageview');

</script>
<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-45985961-1', 'solomidnetwork.com');
ga('send', 'pageview');

</script>
<div id="fb-root"></div>
<script>(function(d, s, id) {
  var js, fjs = d.getElementsByTagName(s)[0];
  if (d.getElementById(id)) return;
  js = d.createElement(s); js.id = id;
  js.src = "//connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.0";
  fjs.parentNode.insertBefore(js, fjs);
}(document, 'script', 'facebook-jssdk'));</script>
</div></body>
</html>

<script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>

  <script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
  <script>
  [].forEach.call(document.querySelectorAll('.adsbygoogle'), function(){
    (adsbygoogle = window.adsbygoogle || []).push({});
  });
  </script>
```

Other returns I've seen are things like HTML being mixed in with the objects within a script tag, sometimes all the script tags missing, ect. Again, the above script never fails if run outside of electron using Node 4.1.0, or using electron 0.30.5/iojs 2.1.3

**Comment by zcbenz** [comment]
I can reproduce in v0.33.1, and after testing I believe it has bene fixed by #2875. You have to wait for a new release for the fix though.

**Comment by dustinblackman** [comment]
Appreciate the reply, thanks!
